megabus/src/main/java/com/bazaarvoice/megabus/refproducer/MegabusRefProducer.java

```diff
@@ -83,6 +89,8 @@ public MegabusRefProducer(MegabusRefProducerConfiguration config, DatabusEventSt
         _executor = executor;
         _producer = requireNonNull(producer, "producer");
         _clock = firstNonNull(clock, Clock.systemUTC());
+        _eventCounter = metricRegistry.counter(MetricRegistry.name("bv.emodb.megabus", "MegabusRefProducer", "event-count"));
```

Comment:
It _is_ more accurate, but in a different way. `1MinuteRate` uses a decaying reservoir for its distribution of values, resulting in an emphasis on more recent values. Datadog's `rate` will use all of the data in the window (whatever window is chosen), but it performs early aggregations that smooth over the data (sort of like a rounding error). I wasn't taking my own advice and going for the fewest possible metrics, especially because I don't know what it would mean to divide one `1MinuteRate` by another. I guess the computation is associative, though. I'll swap these counters to meters. Thanks.